tests/playwright/tests/BuyTon.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { test, expect } from '@playwright/test';
+test.setTimeout(4 * 60 * 1000);
+//Buy TON flow on main screen
+
+test('Buy TON on main screen', async ({ page }) => {
+  await page.goto('/');
+  await page.getByRole('button', { name: 'Get started' }).click();
+  await page.getByRole('button', { name: 'Existing Wallet Import wallet' }).click();
+  await page.getByLabel('1:', { exact: true }).click();
+  await page.getByLabel('1:', { exact: true }).fill(process.env.TON_MNEMONIC_24);
+  await page.getByRole('button', { name: 'Continue' }).click();
+  await page.getByRole('button', { name: 'Continue' }).click();
+  await page
+    .locator('div')
+    .filter({ hasText: /^Password$/ })
+    .getByRole('textbox')
+    .fill('123456');
+  await page.getByRole('textbox').nth(1).click();
+  await page.getByRole('textbox').nth(1).fill('123456');
+  await page.getByRole('button', { name: 'Continue' }).click();
+  await page.getByRole('button', { name: 'Save' }).click();
+  await page.getByRole('button', { name: 'Buy' }).click();
+  await expect(page.getByRole('heading', { name: 'Buy TON' })).toBeVisible();
+  await expect(page.getByText('Mercuryo')).toBeVisible();
+  await expect(page.getByText('Instant one-click purchase up')).toBeVisible();
+  await expect(page.locator('.sc-evBeLY').first()).toBeVisible();
+  await expect(page.locator('div').filter({ hasText: /^NeocryptoInstantly buy with a credit card$/ }).nth(2)).toBeVisible();
```

Comment:
The exact items on this list depend of the country,
For test better to check is there something in this list? 
and make an exception if list is empty

Reply:
Agreed. I`ll need help here